src/app_unit.rs

```diff
@@ -453,3 +539,27 @@ fn serialize() {
     let serialized = ron::to_string(&Au(42)).unwrap();
     assert_eq!(ron::from_str(&serialized), Ok(Au(42)));
 }
+
+#[cfg(feature = "num_traits")]
```

Comment:
I think it would be good to expand the tests a bit. There should probably be tests for the unsupported casts for instance.